bundles/net.tourbook/src/net/tourbook/weather/openweathermap/OpenWeatherMapRetriever.java

```diff
@@ -394,23 +482,31 @@ private boolean retrieveHistoricalWeather() {
    @Override
    public boolean retrieveHistoricalWeatherData() {
 
+      // If the user has reached the maximum number of requests per day, we notify
+      // them and abort the retrieval
+      if (!canMakeRequest()) {
+
+         TourLogManager.log_ERROR(Messages.Log_HistoricalWeatherRetriever_003_RetrievalLimitReached);
+         return false;
+      }
```

Comment:
Can you stop the retrieval logging after reaching the limit, othewise the log is filled?

![image](https://github.com/mytourbook/mytourbook/assets/1283445/4cf40d5d-079b-41c8-bb4f-8593d52df03e)

Reply:
Good idea
Fixed

Example: Below, I have selected several tours but only 1 error message is displayed

![image](https://github.com/mytourbook/mytourbook/assets/25235665/7c4240d1-889e-4974-bfa0-05f63eeb551d)